Dockerfile

```diff
@@ -15,14 +17,16 @@ RUN yum install -y epel-release && \
            -e 's|<Directory "/var/www">|<Directory "/usr/share/nagios/html">|' \
            /etc/httpd/conf/httpd.conf && \
     touch /supervisord.log /supervisord.pid && \
-    mkdir -p /var/log/nagios/archives /var/log/nagios/rw/ /var/log/nagios/spool/checkresults && \
+    mkdir -p /var/log/nagios/archives /var/log/nagios/rw/ /var/log/nagios/spool/checkresults $NAGIOS_PLUGINS && \
     chmod -R 777 /supervisord.log /supervisord.pid /var/log/nagios \
                  /etc/httpd /etc/passwd /var/log /etc/nagios /usr/lib64/nagios \
                  /var/spool/nagios /run /usr/share/httpd /usr/share/nagios && \
     sed -i -e 's|cfg_file=/etc/nagios/objects/localhost.cfg||' /etc/nagios/nagios.cfg
 
 ADD supervisord.conf /etc/supervisord.conf
 ADD make-nagios-fhservices-cfg make-nagios-commands-cfg fhservices.cfg.j2 commands.cfg.j2 /opt/rhmap/
+ADD /plugins/default/* /plugins/core/* $NAGIOS_PLUGINS
+RUN chmod -R 755 $NAGIOS_PLUGINS
```

Comment:
@mikenairn here is a good example of where it may make sense to have another RUN instruction (as we have here) instead of combining all into one (as I mentioned elsewhere as a "best practice").
If we were to move this to the RUN instruction above, we'd have to move the ADD also.

Then the "problem" starts to be cache invalidation:

```Dockerfile
FROM ...

ADD /plugins/default/* /plugins/core/* $NAGIOS_PLUGINS
...
RUN yum install ... && \
        chmod -R 755 $NAGIOS_PLUGINS
```

That building that Dockerfile will re-run `yum install` at least as often as files in plugin/default change, and we normally want to avoid that to speed up local builds.

Release builds should be run without cache.